pkg/security/security_profile/dump/manager.go

```diff
@@ -432,7 +432,7 @@ func (adm *ActivityDumpManager) HandleCGroupTracingEvent(event *model.CgroupTrac
 	defer adm.Unlock()
 
 	if len(event.ContainerContext.ContainerID) == 0 {
-		seclog.Errorf("received a cgroup tracing event with an empty container ID")
+		seclog.Warnf("received a cgroup tracing event with an empty container ID")
```

Comment:
I guess this should trigger since the PR introducing cgroupsID.
So, in this state (if I understand correctly) it will continue to trigger warns. May be we should also do another change

Reply:
After discussion: there is a bug to fix elsewhere.
ok to merge the PR like this until the bug is fixed (and the log restored to error)